### PR TITLE
Update title split

### PR DESCRIPTION
**Pull request by jkdrangel**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,7 +167,7 @@ <h2 class="meetup-date">24 nov</h2>
     .then((response) => {
       response.json().then((data) => {
         let jobs = data.map(d => {
-          let splitTag = d.title.includes(' na ') ? ' na ' : ' at '
+          let splitTag = d.title.includes(' na ') ? ' na ' : d.title.includes(' no ') ? ' no ' : ' at '
```
Comment (rafaelverger) [inline index.html]: [String.prototype.split can be used with a regex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split#Removing_spaces_from_a_string):
```suggestion
          const splitTag = /\s+(?:n[ao]|at)\s+/i;
```

File: index.html [hunk 2]
```diff
@@ -201,7 +201,7 @@ <h2 class="meetup-date">24 nov</h2>
     });
 </script>
 <script>
-  fetch('https://api.github.com/repos/devfsa/meetups/issues/3/comments', { headers: { "Accept": "text" } })
+  fetch('https://api.github.com/repos/devfsa/meetups/issues/4/comments', { headers: { "Accept": "text" } })
```
Comment (rafaelverger) [inline index.html]: This is not related with "updating title split". You should fix this in another PR.. perhaps doing some automation that avoid u to update code file every time a meetup is created

**Review by rafaelverger — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (di3goleite) [reply index.html]: Agreed.

File: index.html (hunk 1 above)
Comment (jkdrangel) [reply index.html]: @rafaelverger Regex have a problem with the spaces on the split

File: index.html (hunk 2 above)
Comment (ssscassio) [reply index.html]: **Not related with the PR.**
The endpoint https://api.github.com/repos/devfsa/meetups/issues return all open issues. It can be used to auto update the next meetup id.
However, there is not enough information to fit the left side of the board.
captura de tela 2018-11-29 as 12 36 19
To really automate I think that we should integrate with [Meetups Api](https://www.meetup.com/meetup_api/docs/:urlname/events/#list)

File: index.html (hunk 1 above)
Comment (rafaelverger) [reply index.html]: Yeah... split using regex must ignore matching groups, my bad. I've fixed the regex in the suggestion above from `/ ((na|o)|(at)) /i;` TO `/\s+(?:n[ao]|at)\s+/i;`

Here are the tests:
```js
const re = /\s+(?:n[ao]|at)\s+/i;
console.log('Job no Empresa'.split(re))          // [ 'Job', 'Empresa' ]
console.log('Job at Empresa'.split(re))          // [ 'Job', 'Empresa' ]
console.log('Job na Empresa'.split(re))          // [ 'Job', 'Empresa' ]
console.log('Job aT Empresa X'.split(re))        // [ 'Job', 'Empresa X' ]
console.log('Job NA Empresa X'.split(re))        // [ 'Job', 'Empresa X' ]
console.log('Job de Foo na Empresa X'.split(re)) // [ 'Job de Foo', 'Empresa X' ]
console.log('Job de Foo at Empresa X'.split(re)) // [ 'Job de Foo', 'Empresa X' ]
console.log('Job de Foo No Empresa X'.split(re)) // [ 'Job de Foo', 'Empresa X' ]
```

File: index.html (hunk 2 above)
Comment (rafaelverger) [reply index.html]: If we agree that all issues of meetup editions must have title: `Edição $NUM - $DATEISO` we can do this:
```js
const meetupTitleRE = /edição \d+\s*-\s*\d{2}\/\d{2}\/\d{4}/i;
fetch('https://api.github.com/repos/devfsa/meetups/issues?state=open')
  .then(r => r.json())
  .then(issues => issues.filter(i => i.title.match(meetupTitleRE))[0])
  .then(issue => fetch(issue.comments_url).then(r => r.json()))
  .then(meetupTalks => doStuff(meetupTalks))
```

Or if we define that the standard for meetup talks issues will be some label added to the issue:
```js
fetch('https://api.github.com/repos/devfsa/meetups/issues?state=open')
  .then(r => r.json())
  .then(issues => issues.filter(i => i.labels.some(label => label.name === 'meetupTalks'))[0])
  .then(issue => fetch(issue.comments_url).then(r => r.json()))
  .then(meetupTalks => doStuff(meetupTalks))
```

**Review by rafaelverger — APPROVED** [review]
